plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -444,31 +529,75 @@ private Object coerce(String context, Type type, Object o) throws Exception {
             }
         }
 
-        Object control;
+        Object control = null;
         try {
             control = instantiate(constructorOnlyDataBoundProps);
         } catch (Exception x) {
             LOGGER.log(Level.WARNING, "Cannot create control version of " + type + " using " + constructorOnlyDataBoundProps, x);
-            return r;
         }
 
-        for (DescribableParameter p : parameters.values()) {
-            if (p.isRequired())
-                continue;
+        if (control!=null) {
+            for (DescribableParameter p : parameters.values()) {
+                if (p.isRequired())
+                    continue;
 
-            Object v = p.inspect(control);
+                Object v = p.inspect(control);
 
-            // if the control has the same value as our object, we won't need to keep it
-            if (ObjectUtils.equals(v, r.get(p.getName()))) {
-                r.remove(p.getName());
+                // if the control has the same value as our object, we won't need to keep it
+                if (ObjectUtils.equals(v, r.get(p.getName()))) {
+                    r.remove(p.getName());
+                }
             }
         }
 
-        return r;
+        UninstantiatedDescribable ud = new UninstantiatedDescribable(symbolOf(o), null, r);
+        ud.setModel(this);
+        return ud;
+    }
+
+    /**
+     * Finds a symbol for an instance if there's one, or return null.
+     */
+    /*package*/ static String symbolOf(Object o) {
+        if (o instanceof Describable) {
+            Descriptor d = ((Describable) o).getDescriptor();
+            return symbolOf(d.getClass());
+        }
+        // TODO JENKINS-26093 hack, pending core change
```

Comment:
This would work for uninstantiation. But how could it work for instantiation, at runtime? `resolveClass` would need the corresponding patch I think. Is this tested?